crates/server/src/handlers/get_pricing_data.rs

```diff
@@ -20,41 +26,13 @@ pub struct PitchLakeJobRequestParams {
 pub struct PitchLakeJobRequest {
     identifiers: Vec<String>,
     params: PitchLakeJobRequestParams,
-    callback_url: String,
-}
-
-#[derive(Debug, Deserialize, Serialize)]
-#[serde(untagged)]
-pub enum PitchLakeJobCallback {
-    Fail(PitchLakeJobFailedCallback),
-    Success(PitchLakeJobSuccessCallback),
-}
-
-// TODO: Placeholder for now, need to be more generic
-// Note that all the number fields are f64; this is because
-// json supports only i32 and for some reason f64 when deserializing.
-// If we want values other than these, making them string might be a
-// good idea.
-#[derive(Debug, Deserialize, Serialize)]
-pub struct PitchLakeJobSuccessCallback {
-    pub job_id: String,
-    pub twap: f64,
-    pub volatility: f64,
-    pub reserve_price: f64,
-}
-
-/// TODO: might want to introduce a 'status' field or
-/// an 'error code' field to make error handling on client
-/// side better.
-#[derive(Debug, Deserialize, Serialize)]
-pub struct PitchLakeJobFailedCallback {
-    pub job_id: String,
-    pub error: String,
 }
 
 #[derive(Debug, Deserialize, Serialize)]
 pub struct JobResponse {
     job_id: String,
+    message: String,
+    status_url: String,
```

Comment:
Both `message` & `status_url` are redundant fields, given that `/job_status` already returns the status information, and `message` is a human description, which again should be at either the status endpoint or only available during an error response.

If we insists on including some information in job response, making it a universal Response similar to what we get in job status would be ideal, in which case just returning the status enum instead of the status_url.

```rust
pub struct JobStatusResponse {
    job_id: String,
    message: String,
    status: JobStatus
}
```